eth/api.go

```diff
@@ -289,6 +289,43 @@ func (api *PublicDebugAPI) DumpBlock(blockNr rpc.BlockNumber) (state.Dump, error
 	return stateDb.RawDump(false, false, true), nil
 }
 
+// IteratorAccountMaxResults is the maximum number of results to be returned per request
+const IteratorAccountMaxResults = 1000
+
+// IteratorAccountAt enumerates all accounts in the given block and start point in paging request
+func (api *PublicDebugAPI) IteratorAccountAt(blockNr rpc.BlockNumber, start []byte, maxResults int, nocode, nostorage, incompletes bool) (state.IteratorDump, error) {
```

Comment:
We now have a `rpc.BlockNumberOrHash`, might be better to use that instead of `rpc.BlockNumber`